src/core_plugins/timelion/public/directives/interval/interval.js

```diff
@@ -9,6 +9,7 @@ app.directive('timelionInterval', function ($compile, $timeout) {
     restrict: 'E',
     scope: {
       model: '=', // The interval model
+      inHeader: '=',
```

Comment:
This is used to differentiate between this UI in the context of the Timelion app and the Visualize app (when you create a Timelion vis). I can add a comment about this.